gapic-generator-java/src/main/java/com/google/api/generator/gapic/composer/comment/ServiceClientCommentComposer.java

```diff
@@ -231,15 +232,17 @@ private static String createTableOfMethods(List<MethodAndVariants> methodAndVari
     StringBuilder tableBuilder = new StringBuilder();
     tableBuilder
         .append("<table>\n")
+        .append("   <caption>Methods</caption>\n")
         .append("   <tr>\n")
         .append("     <th>Method</th>\n")
         .append("     <th>Description</th>\n")
-        .append("     <th>Method Variants</th>\n");
+        .append("     <th>Method Variants</th>\n")
+        .append("   </tr>\n");
     for (MethodAndVariants method : methodAndVariantsList) {
       tableBuilder
           .append("   <tr>\n")
           .append("     <td>")
-          .append(method.method)
+          .append(CommentFormatter.formatAsJavaDocComment(method.method, null))
```

Comment:
Let's see if we can refactor `CommentFormatter` so that it can be used for the whole table, otherwise this issue may happen again in the future. That being said, if it turns out to be more complicated that it supposed to, we can merge this for now and create another issue to track the refactoring effort.

Reply:
I investigated this a bit and I think it would be a little tricky to refactor `CommentFormatter` to be able to be applied to a whole table as I think it would need to have changes (or additions) to `JavadocComment` to only escape certain characters if they're not part of an html tag. 

I could rewrite the table in general to use `JavadocComment` instead of `StringBuilder` so at least it would be explicitly clear when we are *not* applying standard formatting (e.g. escaping HTML characters). Let me know if you think that's worth doing or if I should create an issue for the larger refactoring effort.

Reply:
Tracking issue for refactor: https://github.com/googleapis/sdk-platform-java/issues/2385